openapi/src/parts/workspace.yaml

```diff
@@ -545,3 +549,12 @@ components:
       type: string
       enum: ['RAWLS_WORKSPACE', 'MC_WORKSPACE']
 
+    WorkspaceActivityChangeAgent:
+      description: Info for workspace activity change agent
+      required: [userEmail]
+      properties:
+        userEmail:
+          type: string
+          description: user email of the change agent
+        subjectId:
```

Comment:
What is the subjectId, and when could it be empty?

Reply:
Subject ID is sam ID. I don't think we should expose subject ID in API. Client just needs email.

Also, should Dan review this PR?

Reply:
actually it shouldn't be empty.

https://cs.github.com/broadinstitute/sam/blob/3081228a206fe4ad44911bb948af18bf506ab67c/src/main/resources/swagger/tos-enabled.yaml?q=userSubjectId#L95

it's user id stored in SAM per terra user. it equals to either a google subject id or a an azure b2c id (this part seems to still be some on-going work as there's no api exposing it.